blocks/api/parser.js

```diff
@@ -221,4 +221,4 @@ export function parseWithGrammar( content ) {
 	}, [] );
 }
 
-export default parseWithTinyMCE;
+export default parseWithGrammar;
```

Comment:
Shouldn't matter either way, but is this change intentional?

Separately, it might be nice to run the CI build against both parsers.